instagram/src/App.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { PostContainer } from './components/PostContainer';
+import postData from './application-data';
+import LikeButton from './components/LikeButton';
+import NavBar from './components/NavBar';
+import './App.css';
+
+class App extends Component {
+  constructor(props) {
```

Comment:
No need to pass `props` to `constructor` and `super` in this case since you aren't calling `props` inside the constructor.